frontend/lib/modules/about/team.dart

```diff
@@ -0,0 +1,129 @@
+import 'package:flutter/material.dart'; //need on every page
+
+import '../../app_scaffold.dart';
+
+class Team extends StatefulWidget {
+  @override
+  _TeamState createState() => _TeamState();
+}
+
+class _TeamState extends State<Team> {
+  
+  // List of headshots, names, and roles
+  final List<ImageData> imagesData = [
+    ImageData(
+      imagePath: 'assets/images/luke-madera-headshot.jpg',
+      name: 'Luke Madera',
+      role: 'Product Engineer',
+    ), 
+    ImageData(
+      imagePath: 'assets/images/layla-tadjpour-headshot.jpg',
+      name: 'Layla Tadjpour',
+      role: 'Product Engineer',
+    ),
+    ImageData(
+      imagePath: 'assets/images/claire-adair-headshot.jpg',
+      name: 'Claire Adair',
+      role: 'Customer Acquisition',
+    ),
+    ImageData(
+      imagePath: 'assets/images/jacob-russo-headshot.jpg',
+      name: 'Jacob Russo',
+      role: 'Web Developer',
+    ),
+  ];
+
+  @override
+  void initState() {
+    super.initState();
+  } 
+
+  @override
+  Widget build(BuildContext context) {
+    return AppScaffoldComponent(
+      
+      body: Column(
+        children: [
+
+          // Title widget
+          Padding(
+            padding: const EdgeInsets.all(10.0),
+            child: Text(
+              'Team',
+              style: TextStyle(fontSize: 20, fontWeight: FontWeight.bold),
+            ),
+          ),
+
+          // ListView 
+          Expanded(
+            child: Center( // Align center
+              child: ListView(
+                children: [
+                  // For each pair of images a Row is created with 2 Columns of images
```

Comment:
Nice job! In this case I think we can use `Wrap` and get a lot of this "for free" and make it more dynamic (e.g. 1 image per row on narrow screens, and 3 or 4 on wider screens). We will hopefully add more team members, so making it dynamic (or "responsive") is always a good idea!